.autoblocks.yml

```diff
@@ -4,9 +4,13 @@ autogenerate:
 
     prompts:
       - id: used-by-ci-dont-delete
-        major_versions:
-          - 2
+        major_version: 2
 
       - id: used-by-ci-dont-delete-no-params
-        major_versions:
-          - 1
+        major_version: 1
+
+      - id: text-summarization
+        dangerously_use_undeployed_revision: latest
+
+      - id: question-answerer
+        dangerously_use_undeployed_revision: clvodtv700003a2z02fumceby
```

Comment:
what's still a little awkward is that if you want to both autogenerate + use a specific revision you have to specify it here (to generate the classes) and again when initializing the manager (via the minor_version arg), i think can do something nicer in a followup